### ISSUE TITLE
Tensorflow in Js

### ISSUE THREAD

**Issue by akash-pal** [Neha/AMA#2]
Have you tried implementing tensor flow using JavaScript, could you provide some demo materials?

**Comment by Neha** [comment]
Yes, I have done some basic work on Tensorflow.js. As of now, they are not in the condition to share but soon I will be sharing learning, repos, and resources.
